src/main/java/com/n1303/birthdaybuddy/controller/FriendsController.java

```diff
@@ -33,4 +33,10 @@ public ResponseEntity<Friendship> insert(@RequestParam("firstUser") String first
     public ResponseEntity<List<User>> getAllFriendsOfUser(@RequestParam("userId") String userId) {
         return new ResponseEntity<>(this.friendService.getAllUsersFriend(userId), HttpStatus.OK);
     }
+
+    @PostMapping(UrlConstant.DELETE_FRIENDSHIP)
+    public boolean delete(@RequestParam("firstUser") String firstUser,
```

Comment:
Again, would suggest to return this wrapped in a RequestEntity, reason being we can control the return type http code, headers and a lot of stuff